apps/emqx_management/src/emqx_mgmt_api_clients.erl

```diff
@@ -548,11 +550,7 @@ unsubscribe(post, #{bindings := #{clientid := ClientID}, body := TopicInfo}) ->
 subscribe_batch(post, #{bindings := #{clientid := ClientID}, body := TopicInfos}) ->
     Topics =
         [
-            begin
-                Topic = maps:get(<<"topic">>, TopicInfo),
-                Qos = maps:get(<<"qos">>, TopicInfo, 0),
-                #{topic => Topic, qos => Qos}
-            end
+            emqx_map_lib:unsafe_atom_key_map(TopicInfo)
```

Comment:
nit: would it be better to just get the desired value, although the code would look a bit more verbose?

Reply:
Mainly because I believe in swagger, but this is really not safe, add new fields may cause problems in the future